### DISCUSSION TITLE
Disable external link images

### DISCUSSION THREAD

**Discussion by eyesnareinc** [mozilla/hubs#3706] [Hubs Cloud Questions] [answered]
Is there a way to disable visibility for link images to external links in the same way you can disable visibility for local link images by unchecking controls in spoke?

**Comment by robinkwilson** [comment, marked as answer]
Do you mind linking some screenshots of what you mean? Thank you!

**Reply by robinkwilson** [reply]
Marking as answered, unmark answer if you have those screenshots! Thank you!
